pkg/controllers/provisioner/v1alpha1/controller.go

```diff
@@ -43,7 +43,7 @@ func (c *Controller) Owns() []controllers.Object {
 }
 
 func (c *Controller) Interval() time.Duration {
-	return 10 * time.Second
+	return 60 * time.Second
```

Comment:
Are we changing it to 1 minute?

Reply:
Just to avoid overrun until your changes are in.